### ISSUE TITLE
Ability to move, drag-drop and duplicate entity

### ISSUE THREAD

**Issue by gpaar** [pro3d-space/PRo3D#476]
It shall be possible to duplicate move graphical entities (also drag-drop), such as annotations and scale bars. 
Once such a graphical element is selected, e.g. right mouse click or ctrl-c/ctrl-x ctrl-v shall move/duplicate it, with slightly automatic modified ID and in the same group per default. When ctrl-x is used and another folder (e.g. in the surface tree another branch) is selected, it shall be placed there. 
Basically the same look-and-feel as in Windows Explorer.

**Comment by chris-traxler** [comment]
Eventually reuse tree view from EnVis project.
